js/index.js

```diff
@@ -1,55 +1,57 @@
+ /****** Event Listener for Nav *********/
+
+function homePopUp(home){
+    home.style.height = "64px";
+    home.style.width = "64px";
+}    
+function homePopDown(home){
+    home.style.height = "32px";
+    home.style.width = "32px";
+}
 
 const nav = document.querySelectorAll('nav a')
```

Comment:
You're not too far off track here, Fnu.
```suggestion
const nav = document.querySelectorAll('nav')
nav.addEventListener("mouseover", homePopUp(event))
nav.addEventListener("mouseout", homePopDown(event))
```
And then in the functions, change home.style.height and home.style.width to home.target.style.height and home.target.style.width. I believe that should do it or at least get you closer to a solution.